foundations/installations/prerequisites.md

```diff
@@ -44,7 +44,7 @@ Now that you have VirtualBox installed, launch the program. Once open, you shoul
 
 <img style="padding: 0em;" src="https://cdn.statically.io/gh/TheOdinProject/curriculum/5d27ddb08c8cf3c553537deb6156a5c7f7aa1bac/foundations/installations/prerequisites/imgs/00.png" alt="The VirtualBox start screen" title="A new start">
 
-Click on the “New” button to create a virtual operating system. Give it a name of “Xubuntu”, leave the “Machine Folder” as is, set the “Type” to “Linux” and be sure “Version” is set to “Ubuntu (64-bit)”. Continue by pressing “Next”, and choose the following options in the next steps:
+Click on the “New” button to create a virtual operating system. Give it a name of “Xubuntu”, leave the “Machine Folder” as is, set the “Type” to “Linux” and be sure “Version” is set to “Ubuntu (64-bit)”. If the 64-bit option is not present, you'll likely need to <a href="https://www.google.com/search?q=enable+virtualization+windows" target="_blank">enable virtualization in your computer’s BIOS/UEFI settings</a>. Continue by pressing “Next”, and choose the following options in the next steps:
```

Comment:
Is there a specific resource we can link to instead of a link to a google search?

Reply:
Can we use a markdown link instead of html link please?

```
[enable virtualization in your computer’s BIOS/UEFI settings](https://www.google.com/search?q=enable+virtualization+windows)
```

Reply:
> Is there a specific resource we can link to instead of a link to a google search?
I chose that because it's already referenced on the same page at line 75. [This guide](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_deployment_and_administration_guide/sect-troubleshooting-enabling_intel_vt_x_and_amd_v_virtualization_hardware_extensions_in_bios) seems fairly decent and does not have any unrelated info, other options I've found are part of some other guide or are annoying articles (e.g [one](https://bce.berkeley.edu/enabling-virtualization-in-your-pc-bios.html) and [two](https://helpdeskgeek.com/how-to/how-to-enable-virtualization-in-bios-for-intel-and-amd/)).